cpp/src/arrow/ipc/json-internal.cc

```diff
@@ -736,9 +762,10 @@ class JsonSchemaReader {
       // TODO
       *type = date64();
     } else if (type_name == "time") {
-      return GetTimeLike<TimeType>(json_type, type);
+      // TODO: 32-bit vs 64-bit time
```

Comment:
will remove, thanks. Planning to do some cleaning on this module while completing the JSON read/write support